README.md

```diff
@@ -44,9 +44,12 @@ The quickest way to start a single member cluster for development purposes is to
 [Docker images](https://hub.docker.com/r/hazelcast/hazelcast/).
 
 ```bash
-docker run -p 5701:5701 hazelcast/hazelcast:4.1.1
+docker run -p 5701:5701 hazelcast/hazelcast
 ```
 
+This command fetches the latest Hazelcast version. You can find all available tags
+[here](https://hub.docker.com/r/hazelcast/hazelcast/tags).
+
```

Comment:
BTW, I just saw that there is a link at the top, to a similar page. Maybe this sentence is not required at all

Reply:
I just wanted to emphasize the way they can use to use other hazelcast versions. It is also a good practice to do so in Dockerfile.